README.md

```diff
@@ -344,6 +344,15 @@ Or if you have certificate in `PFX` or `PKCS12` format, setting code can be like
 ### Azure Active Directory(Azure AD)
 Azure AD is Microsoft’s multi-tenant, cloud-based directory and identity management service, you can refer to the [System Variables](#system-variables) section for more details.
 
+### AWS Signature v4
```

Comment:
Also, add the new auth scheme `AWS Signature v4` in the `Main Features` section like other Authentication schemes.

Reply:
Done.